save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt

```diff
@@ -147,6 +148,19 @@ val createVulnerabilityView: VFC = VFC {
                             div {
                                 className = ClassName("row")
 
+                                inputTextFormOptional {
```

Comment:
Here we want the field to be required and already pre-filled with our generated id - _SOTV-..._

Reply:
Will be done in later PR, not in this one.

Moreover, right now the id is generated after it is saved to the database as the identifier depends on the database id

Reply:
As far as I remember, we wanted to add a checkbox `Generate identifier`